requirements.txt

```diff
@@ -1 +1 @@
-
+Pillow
```

Comment:
What are these requirements for? Maybe add some documentation to make it easier to understand for an external part